cmd/config.go

```diff
@@ -19,15 +18,16 @@ var configurationCmd = &cobra.Command{
 	Long: `Modifies Odo specific configuration settings within the config file.
 
 Available Parameters:
-UpdateNotification - Controls if an update notification is shown or not (true or false)`,
+UpdateNotification - Controls if an update notification is shown or not (true or false),
+Timeout            - Timeout (in seconds) for openshift server connection check (minimum value is 1 second)`,
```

Comment:
according to `oc` command line flag,

```
--request-timeout='0': The length of time to wait before giving up on a single server request.
Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h). A value of zero means
don't timeout requests.
```

so, we should be able to set  `0` as well so that it `don't timeout requests.`